tests/EventEmitterTest.php

```diff
@@ -13,60 +13,62 @@
 
 use Evenement\EventEmitter;
 use InvalidArgumentException;
+use PHPUnit\Framework\Attributes\BackupGlobals;
+use PHPUnit\Framework\Attributes\BackupStaticProperties;
+use PHPUnit\Framework\Attributes\Before;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\DoesNotPerformAssertions;
 use PHPUnit\Framework\TestCase;
 
+#[CoversClass(EventEmitter::class)]
+#[BackupGlobals(false)]
+#[BackupStaticProperties(false)]
 class EventEmitterTest extends TestCase
 {
     /**
      * @var EventEmitter
      */
-    private $emitter;
+    private EventEmitter $emitter;
```

Comment:
The doc block can be dropped when we use typed properties (or params etc for that matter) and not add any command or array shapes etc to it.